server/config/options.go

```diff
@@ -137,8 +137,9 @@ var DefaultConfig = Config{
 	},
 	Quota: QuotaConfig{
 		Enabled:              false,
-		RateLimit:            1000,        // requests per second
+		RateLimit:            1000,        // requests per second both reads and writes
```

Comment:
Should we reduce the defaults? So that we can start testing it on the preview cluster?

Reply:
Yes, it's disable for now. Will adjust it later.